deployment/v3/mosip/masterdata-loader/install.sh

```
@@ -8,9 +8,123 @@ fi
+            *)
+                echo "Invalid response. Please enter 'yes' or 'no'."
+                echo ""
+                ;;
+        esac
+    fi
+done
+
 echo  WARNING: This need to be executed only once at the begining for masterdata deployment. If reexecuted in a working env this will reset the whole master_data DB tables resulting in data loss.
 echo  Please skip this if masterdata is already uploaded.
-read -p "CAUTION: Do you still want to continue(Y/n)" yn
+read -p "CAUTION: Do you still want to continue?(Y/n)" yn
```

Shellcheck (0.11.0)

[info] read without -r will mangle backslashes.

(SC2162)